src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -184,6 +184,11 @@ public void onEnable() {
 			TownyLogger.log.info("[Towny] Version: " + version + " - Mod Enabled");
 		TownyLogger.log.info("=============================================================");
 
+		//Add our chat handler to TheNewChat via the API.
+		if(Bukkit.getPluginManager().isPluginEnabled("TheNewChat")) {
```

Comment:
This should probably go into the CheckPlugins() further down for consistency's sake.